Formula/parquet-tools.rb

```diff
@@ -0,0 +1,22 @@
+class ParquetTools < Formula
+  desc "ApacheParquet command-line tools and utilities"
+  homepage "https://parquet.apache.org/"
+  url "https://github.com/apache/parquet-mr.git", :tag => "apache-parquet-1.8.1", :revision => "4aba4dae7bb0d4edbcf7923ae1339f28fd3f7fcf"
```

Comment:
Can you throw this onto three lines so it doesn't stretch the formula out unnecessarily? Thanks!

```ruby
url "https://github.com/apache/parquet-mr.git",
    :tag => "apache-parquet-1.8.1",
    :revision => "4aba4dae7bb0d4edbcf7923ae1339f28fd3f7fcf"
```